main/conceptual/zoe/contracts/vault.md

```diff
@@ -111,13 +111,3 @@ The Vault's public API includes `makeVaultInvitation()` and
 to the issuer of `IST` so anyone can hold, spend and recognize IST.
 `makeVaultInvitation()` is described above under [Borrowers](#borrowers)
 
-### Roadmap
-
- * Finer control over liquidation: we think liquidation should be sensitive to
-  market depth and trade volumes.
- * Vaults in individual Vats: for better isolation and security, we want to put
-  each vault in a separate vat.
- * Limits on IST issued per collateral type: limiting the amount of IST that
-   can be issued for each collateral might blunt some attacks
- * Adding new collateral types will be made subject to governance rather
-   than being included in the creatorFacet.
```

Comment:
Taking these "future directions" lists out makes sense to me... @Chris-Hibbert do you agree?

If they're worth tracking, issues in the agoric-sdk repo make more sense, to me.

Reply:
Yes, makes sense. while these are actually relatively high-priority product directions, it's probably a bad idea to talk about product directions in the write-up for each component. In most places, they'll rapidly become stale.

Reply:
I thought I had removed that; it doesn't exist in my local copy. I'm not sure what happened. In theory, when I push my additional revisions to the cloud, this roadmap will go away.

Reply:
You did remove it; that's why it's red. I just wanted to confirm with Chris that it _should_ be removed.